python/tank/bootstrap/manager.py

```diff
@@ -221,11 +244,50 @@ def set_progress_callback(self, callback):
         """
         self._progress_cb = callback
 
-    def bootstrap_engine(self, engine_name, entity=None):
+    def bootstrap_engine(self,
```

Comment:
Out of curiosity, is there any reason to overload this method and use a property rather than introducing a new method called `async_bootstrap_engine`?